tools/nl/embeddings/data/autogen_input/medium/autogen_svs.csv

```diff
@@ -4768,6 +4753,51 @@ Sales_Establishment_ManufacturerSalesBranchesAndOffices_NAICSWholesaleTrade_With
 Sales_Establishment_NAICSAccommodationFoodServices_WithPayroll,"Sales of Establishment: Accommodation And Food Services (NAICS/72), With Payroll",Sales of Accommodation and Food Services Establishments With Payroll,,
 Sales_Establishment_NAICSWholesaleTrade_WithPayroll,"Sales of Establishment: Wholesale Trade (NAICS/42), With Payroll",Wholesale Trade Establishments With Payrolls,,
 Sales_Establishment_USC_MerchantWholesalers_NAICSWholesaleTrade_WithPayroll,"Sales of Establishment: Merchant Wholesalers, Wholesale Trade (NAICS/42)",Wholesale Trade Merchant Wholesalers,,
+sdg/AG_FOOD_WST,Food waste,,,
+sdg/AG_FOOD_WST_PC,Food waste per capita,,,
+sdg/AG_LND_FRST,Forest area as a proportion of total land area,,,
+sdg/AG_LND_FRSTBIOPHA,Above-ground biomass in forest,,,
+sdg/AG_LND_FRSTCERT,Forest area under an independently verified forest management certification scheme,,,
+sdg/AG_LND_FRSTCHG,Annual forest area change rate,,,
+sdg/AG_LND_FRSTN,Forest area,,,
+sdg/AG_LND_FRSTPRCT,Proportion of forest area within legally established protected areas,,,
+sdg/DC_ODA_BDVDL,Total official development assistance for biodiversity,,,
+sdg/EG_FEC_RNEW,Renewable energy share in the total final energy consumption,,,
```

Comment:
Hmm, why are these being added to medium index?

Reply:
Good question. So they were added a long while back: https://github.com/datacommonsorg/website/commit/b74b19d22645a1a4e5dfc73622da36b616114985

But looks like what happened is that I might have forgotten to copy/paste the autogen file from prep/ to under the autogen_input/medium/